### PR TITLE
Feature/wizard

### PR DESCRIPTION
**Pull request by raxkumar-wedaa**
* Added wizard
* minor features added

### CODE REVIEW

File: src/pages/Rewiew/Infrastructure.js [hunk 1]
```diff
@@ -762,9 +758,10 @@ const Infrastructure = ({ onSubmit, projectData, generateZip }) => {
                 onClick={() => {
                     selectedImage !== 'none' ? handleSubmit(DeploymentData) : onSubmit(projectData, true);
                 }}
-                isDisabled={!selectedImage || isCheckEmpty() || checkValidation()}
+                minH={'48px'}
+                isDisabled={selectedImage !== 'none' && (!selectedImage || isCheckEmpty() || checkValidation())}
             >
-                Generate with Infrastructure
+                Generate
```
Comment (raxkumar-wedaa) [inline src/pages/Rewiew/Infrastructure.js:764]: Generate Code

File: src/components/Navbar.js [hunk 2]
```diff
@@ -52,12 +52,17 @@ export default function Header({ children }) {
...
+                        <Text fontSize="md" color={color} fontWeight={location.pathname === '/canvasToCode' ? 'bold' : 'normal'}>
+                            CanvasToCode
+                        </Text>
+                    </Link>
+                    <Link to="/wizard">
                         <Text
                             fontSize="md"
                             color={color}
-                            fontWeight={location.pathname === '/canvasToCode' || location.pathname === '/' ? 'bold' : 'normal'}
+                            fontWeight={location.pathname === '/wizard' || location.pathname === '/' ? 'bold' : 'normal'}
```
Comment (raxkumar-wedaa) [inline src/components/Navbar.js:63]: Clicking on CanvasToCode opens up 2 options
 
Basic or Advanced
 
Selecting Basic will take the user to Wizard
 
Advanced will directly land on Canvas

File: src/pages/Wizad/index.js [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    useToast,
+} from '@chakra-ui/react';
+import './index.css';
+import WizardBox from './WizardBox';
+import { ArrowBackIcon, ArrowForwardIcon, CheckIcon, CloseIcon, ExternalLinkIcon } from '@chakra-ui/icons';
+import ReviewBox from './ReviewBox';
+
+import { idMappings, questionsData } from './CONSTANTS';
+import { useHistory } from 'react-router-dom/cjs/react-router-dom.min';
+import { dummydata } from '../../test-remove';
```
Comment (raxkumar-wedaa) [inline src/pages/Wizad/index.js:23]: Remove this import if it not required any more!

**Review by raxkumar-wedaa — COMMENTED** [review]
Resolve the review comments

**Review by craxkumar — APPROVED** [review]
Review OK!

**Pull request merged**